roxie/roxiemem/roxiemem.cpp

```diff
@@ -5113,7 +5112,7 @@ class CDataBufferManager : public CInterface, implements IDataBufferManager
                 {
                     curBlock->Release();
                     curBlock = NULL;
-                    nextAddr = NULL;
+                    nextBase = NULL;
```

Comment:
I was lazy - it didn't need clearing.  A comment would be good though - or clear it.